common/client-libs/validator-client/Cargo.toml

```diff
@@ -31,7 +31,7 @@ prost = { version = "0.9", default-features = false, optional = true }
 flate2 = { version = "1.0.20", optional = true }
 sha2 = { version = "0.9.5", optional = true }
 itertools = { version = "0.10", optional = true }
-cosmwasm-std = { git = "https://github.com/jstuczyn/cosmwasm", branch="0.14.1-updatedk256", optional = true }
+cosmwasm-std = { git = "https://github.com/CosmWasm/cosmwasm", rev="v1.0.0-beta2", optional = true }
```

Comment:
`v1.0.0-beta2` is indeed a tag, but to pull tags in `Cargo.toml`, you use keyword `rev` : )
Also I couldn't get it from crates.io because they haven't released it on crates.io

Reply:
There is also a `tag` keyword, but if it works with `rev` as well it's all good. There is also the official release and it seems to work alright: https://crates.io/crates/cosmwasm-std/1.0.0-beta2

Reply:
Huh, For some reason I didn't see it last time... Let me fix that then, good catch!